### PR TITLE
Team feature documentation

### PR DESCRIPTION
**Pull request by Tarun-Dubey**
Added files pertaining to Team feature documentation

### CODE REVIEW

**Comment by rnaresh** [comment]
**Running:  check-git-navs.js**                                                               (3 of 3)
2020-09-16T05:59:14.916Z [INFO]: 
2020-09-16T05:59:14.918Z [INFO]: 
2020-09-16T05:59:14.925Z [INFO]: Check for nav
2020-09-16T05:59:19.772Z [INFO]: 1) Verifies left nav in: teams/code-repository
2020-09-16T05:59:24.459Z [INFO]: 2) Verifies left nav in: teams/dashboard
2020-09-16T05:59:29.210Z [INFO]: 3) Verifies left nav in: teams/manage-projects
2020-09-16T05:59:33.940Z [INFO]: 4) Verifies left nav in: teams/manage-roles
2020-09-16T05:59:38.655Z [INFO]: 5) Verifies left nav in: teams/manage-subscription
2020-09-16T05:59:43.440Z [INFO]: 6) Verifies left nav in: teams/manage-users
2020-09-16T05:59:48.182Z [INFO]: 7) Verifies left nav in: teams/team-profile
2020-09-16T05:59:48.228Z [INFO]: 
2020-09-16T05:59:48.243Z [INFO]: 
2020-09-16T05:59:48.244Z [INFO]: 0 passing (33s)
                                 7 failing
                                 1) Check for nav
                                 Verifies left nav in: teams/code-repository:
                                 Left Nav is missing in teams/code-repository.md
                                 CypressError: Timed out retrying: expected '[ <nav.slidingNav>, 1 more... ]' to have class 'toc'
                                 at Object.cypressErr (http://localhost:3000/__cypress/runner/cypress_runner.js:86207:11)
                                 at Object.throwErr (http://localhost:3000/__cypress/runner/cypress_runner.js:86162:18)
                                 at Object.throwErrByPath (http://localhost:3000/__cypress/runner/cypress_runner.js:86194:17)
                                 at retry (http://localhost:3000/__cypress/runner/cypress_runner.js:76849:16)
                                 at onFailFn (http://localhost:3000/__cypress/runner/cypress_runner.js:65634:16)
                                 at tryCatcher (http://localhost:3000/__cypress/runner/cypress_runner.js:120203:23)
                                 at Promise._settlePromiseFromHandler (http://localhost:3000/__cypress/runner/cypress_runner.js:118139:31)
                                 at Promise._settlePromise (http://localhost:3000/__cypress/runner/cypress_runner.js:118196:18)
                                 at Promise._settlePromise0 (http://localhost:3000/__cypress/runner/cypress_runner.js:118241:10)
                                 at Promise._settlePromises (http://localhost:3000/__cypress/runner/cypress_runner.js:118316:18)
                                 at Async../node_modules/bluebird/js/release/async.js.Async._drainQueue (http://localhost:3000/__cypress/runner/cypress_runner.js:114928:16)
                                 at Async../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:3000/__cypress/runner/cypress_runner.js:114938:10)
                                 at Async.drainQueues (http://localhost:3000/__cypress/runner/cypress_runner.js:114812:14)
                                 2) Check for nav
                                 Verifies left nav in: teams/dashboard:
                                 Left Nav is missing in teams/dashboard.md
                                 CypressError: Timed out retrying: expected '[ <nav.slidingNav>, 1 more... ]' to have class 'toc'
                                 at Object.cypressErr (http://localhost:3000/__cypress/runner/cypress_runner.js:86207:11)
                                 at Object.throwErr (http://localhost:3000/__cypress/runner/cypress_runner.js:86162:18)
                                 at Object.throwErrByPath (http://localhost:3000/__cypress/runner/cypress_runner.js:86194:17)
                                 at retry (http://localhost:3000/__cypress/runner/cypress_runner.js:76849:16)
                                 at onFailFn (http://localhost:3000/__cypress/runner/cypress_runner.js:65634:16)
                                 at tryCatcher (http://localhost:3000/__cypress/runner/cypress_runner.js:120203:23)
                                 at Promise._settlePromiseFromHandler (http://localhost:3000/__cypress/runner/cypress_runner.js:118139:31)
                                 at Promise._settlePromise (http://localhost:3000/__cypress/runner/cypress_runner.js:118196:18)
                                 at Promise._settlePromise0 (http://localhost:3000/__cypress/runner/cypress_runner.js:118241:10)
                                 at Promise._settlePromises (http://localhost:3000/__cypress/runner/cypress_runner.js:118316:18)
                                 at Async../node_modules/bluebird/js/release/async.js.Async._drainQueue (http://localhost:3000/__cypress/runner/cypress_runner.js:114928:16)
                                 at Async../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:3000/__cypress/runner/cypress_runner.js:114938:10)
                                 at Async.drainQueues (http://localhost:3000/__cypress/runner/cypress_runner.js:114812:14)
                                 3) Check for nav
                                 Verifies left nav in: teams/manage-projects:
                                 Left Nav is missing in teams/manage-projects.md
                                 CypressError: Timed out retrying: expected '[ <nav.slidingNav>, 1 more... ]' to have class 'toc'
                                 at Object.cypressErr (http://localhost:3000/__cypress/runner/cypress_runner.js:86207:11)
                                 at Object.throwErr (http://localhost:3000/__cypress/runner/cypress_runner.js:86162:18)
                                 at Object.throwErrByPath (http://localhost:3000/__cypress/runner/cypress_runner.js:86194:17)
                                 at retry (http://localhost:3000/__cypress/runner/cypress_runner.js:76849:16)
                                 at onFailFn (http://localhost:3000/__cypress/runner/cypress_runner.js:65634:16)
                                 at tryCatcher (http://localhost:3000/__cypress/runner/cypress_runner.js:120203:23)
                                 at Promise._settlePromiseFromHandler (http://localhost:3000/__cypress/runner/cypress_runner.js:118139:31)
                                 at Promise._settlePromise (http://localhost:3000/__cypress/runner/cypress_runner.js:118196:18)
                                 at Promise._settlePromise0 (http://localhost:3000/__cypress/runner/cypress_runner.js:118241:10)
                                 at Promise._settlePromises (http://localhost:3000/__cypress/runner/cypress_runner.js:118316:18)
                                 at Async../node_modules/bluebird/js/release/async.js.Async._drainQueue (http://localhost:3000/__cypress/runner/cypress_runner.js:114928:16)
                                 at Async../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:3000/__cypress/runner/cypress_runner.js:114938:10)
                                 at Async.drainQueues (http://localhost:3000/__cypress/runner/cypress_runner.js:114812:14)
                                 4) Check for nav
                                 Verifies left nav in: teams/manage-roles:
                                 Left Nav is missing in teams/manage-roles.md
                                 CypressError: Timed out retrying: expected '[ <nav.slidingNav>, 1 more... ]' to have class 'toc'
                                 at Object.cypressErr (http://localhost:3000/__cypress/runner/cypress_runner.js:86207:11)
                                 at Object.throwErr (http://localhost:3000/__cypress/runner/cypress_runner.js:86162:18)
                                 at Object.throwErrByPath (http://localhost:3000/__cypress/runner/cypress_runner.js:86194:17)
                                 at retry (http://localhost:3000/__cypress/runner/cypress_runner.js:76849:16)
                                 at onFailFn (http://localhost:3000/__cypress/runner/cypress_runner.js:65634:16)
                                 at tryCatcher (http://localhost:3000/__cypress/runner/cypress_runner.js:120203:23)
                                 at Promise._settlePromiseFromHandler (http://localhost:3000/__cypress/runner/cypress_runner.js:118139:31)
                                 at Promise._settlePromise (http://localhost:3000/__cypress/runner/cypress_runner.js:118196:18)
                                 at Promise._settlePromise0 (http://localhost:3000/__cypress/runner/cypress_runner.js:118241:10)
                                 at Promise._settlePromises (http://localhost:3000/__cypress/runner/cypress_runner.js:118316:18)
                                 at Async../node_modules/bluebird/js/release/async.js.Async._drainQueue (http://localhost:3000/__cypress/runner/cypress_runner.js:114928:16)
                                 at Async../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:3000/__cypress/runner/cypress_runner.js:114938:10)
                                 at Async.drainQueues (http://localhost:3000/__cypress/runner/cypress_runner.js:114812:14)
                                 5) Check for nav
                                 Verifies left nav in: teams/manage-subscription:
                                 Left Nav is missing in teams/manage-subscription.md
                                 CypressError: Timed out retrying: expected '[ <nav.slidingNav>, 1 more... ]' to have class 'toc'
                                 at Object.cypressErr (http://localhost:3000/__cypress/runner/cypress_runner.js:86207:11)
                                 at Object.throwErr (http://localhost:3000/__cypress/runner/cypress_runner.js:86162:18)
                                 at Object.throwErrByPath (http://localhost:3000/__cypress/runner/cypress_runner.js:86194:17)
                                 at retry (http://localhost:3000/__cypress/runner/cypress_runner.js:76849:16)
                                 at onFailFn (http://localhost:3000/__cypress/runner/cypress_runner.js:65634:16)
                                 at tryCatcher (http://localhost:3000/__cypress/runner/cypress_runner.js:120203:23)
                                 at Promise._settlePromiseFromHandler (http://localhost:3000/__cypress/runner/cypress_runner.js:118139:31)
                                 at Promise._settlePromise (http://localhost:3000/__cypress/runner/cypress_runner.js:118196:18)
                                 at Promise._settlePromise0 (http://localhost:3000/__cypress/runner/cypress_runner.js:118241:10)
                                 at Promise._settlePromises (http://localhost:3000/__cypress/runner/cypress_runner.js:118316:18)
                                 at Async../node_modules/bluebird/js/release/async.js.Async._drainQueue (http://localhost:3000/__cypress/runner/cypress_runner.js:114928:16)
                                 at Async../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:3000/__cypress/runner/cypress_runner.js:114938:10)
                                 at Async.drainQueues (http://localhost:3000/__cypress/runner/cypress_runner.js:114812:14)
                                 6) Check for nav
                                 Verifies left nav in: teams/manage-users:
                                 Left Nav is missing in teams/manage-users.md
                                 CypressError: Timed out retrying: expected '[ <nav.slidingNav>, 1 more... ]' to have class 'toc'
                                 at Object.cypressErr (http://localhost:3000/__cypress/runner/cypress_runner.js:86207:11)
                                 at Object.throwErr (http://localhost:3000/__cypress/runner/cypress_runner.js:86162:18)
                                 at Object.throwErrByPath (http://localhost:3000/__cypress/runner/cypress_runner.js:86194:17)
                                 at retry (http://localhost:3000/__cypress/runner/cypress_runner.js:76849:16)
                                 at onFailFn (http://localhost:3000/__cypress/runner/cypress_runner.js:65634:16)
                                 at tryCatcher (http://localhost:3000/__cypress/runner/cypress_runner.js:120203:23)
                                 at Promise._settlePromiseFromHandler (http://localhost:3000/__cypress/runner/cypress_runner.js:118139:31)
                                 at Promise._settlePromise (http://localhost:3000/__cypress/runner/cypress_runner.js:118196:18)
                                 at Promise._settlePromise0 (http://localhost:3000/__cypress/runner/cypress_runner.js:118241:10)
                                 at Promise._settlePromises (http://localhost:3000/__cypress/runner/cypress_runner.js:118316:18)
                                 at Async../node_modules/bluebird/js/release/async.js.Async._drainQueue (http://localhost:3000/__cypress/runner/cypress_runner.js:114928:16)
                                 at Async../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:3000/__cypress/runner/cypress_runner.js:114938:10)
                                 at Async.drainQueues (http://localhost:3000/__cypress/runner/cypress_runner.js:114812:14)
                                 7) Check for nav
                                 Verifies left nav in: teams/team-profile:
                                 Left Nav is missing in teams/team-profile.md
                                 CypressError: Timed out retrying: expected '[ <nav.slidingNav>, 1 more... ]' to have class 'toc'
                                 at Object.cypressErr (http://localhost:3000/__cypress/runner/cypress_runner.js:86207:11)
                                 at Object.throwErr (http://localhost:3000/__cypress/runner/cypress_runner.js:86162:18)
                                 at Object.throwErrByPath (http://localhost:3000/__cypress/runner/cypress_runner.js:86194:17)
                                 at retry (http://localhost:3000/__cypress/runner/cypress_runner.js:76849:16)
                                 at onFailFn (http://localhost:3000/__cypress/runner/cypress_runner.js:65634:16)
                                 at tryCatcher (http://localhost:3000/__cypress/runner/cypress_runner.js:120203:23)
                                 at Promise._settlePromiseFromHandler (http://localhost:3000/__cypress/runner/cypress_runner.js:118139:31)
                                 at Promise._settlePromise (http://localhost:3000/__cypress/runner/cypress_runner.js:118196:18)
                                 at Promise._settlePromise0 (http://localhost:3000/__cypress/runner/cypress_runner.js:118241:10)
                                 at Promise._settlePromises (http://localhost:3000/__cypress/runner/cypress_runner.js:118316:18)
                                 at Async../node_modules/bluebird/js/release/async.js.Async._drainQueue (http://localhost:3000/__cypress/runner/cypress_runner.js:114928:16)
                                 at Async../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:3000/__cypress/runner/cypress_runner.js:114938:10)
                                 at Async.drainQueues (http://localhost:3000/__cypress/runner/cypress_runner.js:114812:14)
2020-09-16T05:59:48.250Z [INFO]: [mochawesome] Report JSON saved to /codebuild/output/src561133977/src/docs/website/cypress/report/mochawesome-report/mochawesome_09162020_055948.json
2020-09-16T05:59:49.252Z [INFO]: 
2020-09-16T05:59:49.252Z [INFO]: (Results)
2020-09-16T05:59:49.255Z [INFO]: 
2020-09-16T05:59:49.257Z [INFO]: ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
                                 │ Tests:        7                                                                                │
                                 │ Passing:      0                                                                                │
                                 │ Failing:      7                                                                                │
                                 │ Pending:      0                                                                                │
                                 │ Skipped:      0                                                                                │
                                 │ Screenshots:  7                                                                                │
                                 │ Video:        true                                                                             │
                                 │ Duration:     33 seconds                                                                       │
                                 │ Spec Ran:     check-git-navs.js                                                                │
                                 └────────────────────────────────────────────────────────────────────────────────────────────────┘
2020-09-16T05:59:49.258Z [INFO]: (Screenshots)
2020-09-16T05:59:49.260Z [INFO]: -  /codebuild/output/src561133977/src/docs/website/cypress/screenshots/check-git-na     (1280x720)
                                 vs.js/Check for nav -- Verifies left nav in teamscode-repository (failed).png
                                 -  /codebuild/output/src561133977/src/docs/website/cypress/screenshots/check-git-na     (1280x720)
                                 vs.js/Check for nav -- Verifies left nav in teamsdashboard (failed).png
                                 -  /codebuild/output/src561133977/src/docs/website/cypress/screenshots/check-git-na     (1280x720)
                                 vs.js/Check for nav -- Verifies left nav in teamsmanage-projects (failed).png
                                 -  /codebuild/output/src561133977/src/docs/website/cypress/screenshots/check-git-na     (1280x720)
                                 vs.js/Check for nav -- Verifies left nav in teamsmanage-roles (failed).png
                                 -  /codebuild/output/src561133977/src/docs/website/cypress/screenshots/check-git-na     (1280x720)
                                 vs.js/Check for nav -- Verifies left nav in teamsmanage-subscription (failed).pn
                                 g
                                 -  /codebuild/output/src561133977/src/docs/website/cypress/screenshots/check-git-na     (1280x720)
                                 vs.js/Check for nav -- Verifies left nav in teamsmanage-users (failed).png
                                 -  /codebuild/output/src561133977/src/docs/website/cypress/screenshots/check-git-na     (1280x720)
                                 vs.js/Check for nav -- Verifies left nav in teamsteam-profile (failed).png
2020-09-16T05:59:49.286Z [INFO]: 
2020-09-16T05:59:49.286Z [INFO]: (Video)
2020-09-16T05:59:49.286Z [INFO]: -  Started processing:  Compressing to 32 CRF
2020-09-16T05:59:53.753Z [INFO]: -  Finished processing: /codebuild/output/src561133977/src/docs/website/cypress/vid    (4 seconds)
                                 eos/check-git-navs.js.mp4
2020-09-16T05:59:53.755Z [INFO]: 
2020-09-16T05:59:53.757Z [INFO]: 
2020-09-16T05:59:53.777Z [WARNING]: tput: No value for $TERM and no -T specified
2020-09-16T05:59:53.778Z [INFO]: ================================================================================
                                 (Run Finished)
2020-09-16T05:59:53.779Z [INFO]: 
2020-09-16T05:59:53.782Z [INFO]: Spec                                              Tests  Passing  Failing  Pending  Skipped
                                 ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
                                 │ ✔  basic_spec.js                            00:04        4        4        -        -        - │
                                 ├────────────────────────────────────────────────────────────────────────────────────────────────┤
                                 │ ✔  check-git-anchor.js                      00:33        7        7        -        -        - │
                                 ├────────────────────────────────────────────────────────────────────────────────────────────────┤
                                 │ ✖  check-git-navs.js                        00:33        7        -        7        -        - │
                                 └────────────────────────────────────────────────────────────────────────────────────────────────┘
                                 ✖  1 of 3 failed (33%)                      01:11       18       11        7        -        -
2020-09-16T05:59:53.836Z [INFO]: [?25h
2020-09-16T05:59:53.841Z [WARNING]: npm
2020-09-16T05:59:53.842Z [WARNING]: ERR! code ELIFECYCLE
                                    npm ERR! errno 7
2020-09-16T05:59:53.842Z [WARNING]: npm
2020-09-16T05:59:53.844Z [WARNING]: ERR! @ cypress:prod: `cypress run --reporter mochawesome --reporter-options 'reportDir=cypress/report/mochawesome-report,overwrite=false,html=false,json=true,timestamp=mmddyyyy_HHMMss'`
                                    npm ERR! Exit status 7
                                    npm ERR!
                                    npm ERR! Failed at the @ cypress:prod script.
                                    npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
2020-09-16T05:59:53.848Z [WARNING]: 
2020-09-16T05:59:53.848Z [WARNING]: npm ERR! A complete log of this run can be found in:
                                    npm ERR!     /root/.npm/_logs/2020-09-16T05_59_53_843Z-debug.log
2020-09-16T05:59:53.848Z [HELP]: Outputting the npm debug log
                                 0 info it worked if it ends with ok
                                 1 verbose cli [ '/root/.nvm/versions/node/v10.16.0/bin/node',
                                 1 verbose cli   '/root/.nvm/versions/node/v10.16.0/bin/npm',
                                 1 verbose cli   'run',
                                 1 verbose cli   'cypress:prod' ]
                                 2 info using npm@6.9.0
                                 3 info using node@v10.16.0
                                 4 verbose run-script [ 'precypress:prod', 'cypress:prod', 'postcypress:prod' ]
                                 5 info lifecycle @~precypress:prod: @
                                 6 info lifecycle @~cypress:prod: @
                                 7 verbose lifecycle @~cypress:prod: unsafe-perm in lifecycle true
                                 8 verbose lifecycle @~cypress:prod: PATH: /root/.nvm/versions/node/v10.16.0/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/codebuild/output/src561133977/src/docs/website/node_modules/.bin:/usr/local/rvm/gems/ruby-2.4.6/bin:/usr/local/rvm/gems/ruby-2.4.6@global/bin:/usr/local/rvm/rubies/ruby-2.4.6/bin:/usr/local/rvm/bin:/root/.yarn/bin:/root/.config/yarn/global/node_modules/.bin:/root/.nvm/versions/node/v10.16.0/bin:/root/.local/bin:/usr/local/rvm/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
                                 9 verbose lifecycle @~cypress:prod: CWD: /codebuild/output/src561133977/src/docs/website
                                 10 silly lifecycle @~cypress:prod: Args: [ '-c',
                                 10 silly lifecycle   'cypress run --reporter mochawesome --reporter-options \'reportDir=cypress/report/mochawesome-report,overwrite=false,html=false,json=true,timestamp=mmddyyyy_HHMMss\'' ]
                                 11 silly lifecycle @~cypress:prod: Returned: code: 7  signal: null
                                 12 info lifecycle @~cypress:prod: Failed to exec cypress:prod script
                                 13 verbose stack Error: @ cypress:prod: `cypress run --reporter mochawesome --reporter-options 'reportDir=cypress/report/mochawesome-report,overwrite=false,html=false,json=true,timestamp=mmddyyyy_HHMMss'`
                                 13 verbose stack Exit status 7
                                 13 verbose stack     at EventEmitter.<anonymous> (/root/.nvm/versions/node/v10.16.0/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:301:16)
                                 13 verbose stack     at EventEmitter.emit (events.js:198:13)
                                 13 verbose stack     at ChildProcess.<anonymous> (/root/.nvm/versions/node/v10.16.0/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
                                 13 verbose stack     at ChildProcess.emit (events.js:198:13)
                                 13 verbose stack     at maybeClose (internal/child_process.js:982:16)
                                 13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:259:5)
                                 14 verbose pkgid @
                                 15 verbose cwd /codebuild/output/src561133977/src/docs/website
                                 16 verbose Linux 4.14.193-113.317.amzn1.x86_64
                                 17 verbose argv "/root/.nvm/versions/node/v10.16.0/bin/node" "/root/.nvm/versions/node/v10.16.0/bin/npm" "run" "cypress:prod"
                                 18 verbose node v10.16.0
                                 19 verbose npm  v6.9.0
                                 20 error code ELIFECYCLE
                                 21 error errno 7
                                 22 error @ cypress:prod: `cypress run --reporter mochawesome --reporter-options 'reportDir=cypress/report/mochawesome-report,overwrite=false,html=false,json=true,timestamp=mmddyyyy_HHMMss'`
                                 22 error Exit status 7
                                 23 error Failed at the @ cypress:prod script.
                                 23 error This is probably not a problem with npm. There is likely additional logging output above.
                                 24 verbose exit [ 7, true ]
2020-09-16T05:59:53.849Z [INFO]: # Completed phase: test
2020-09-16T05:59:53.850Z [INFO]: # Starting phase: postTest
                                 # Executing command: npm run cypress:prep-report
2020-09-16T05:59:54.024Z [INFO]: > @ cypress:prep-report /codebuild/output/src561133977/src/docs/website
                                 > npx mochawesome-merge --reportDir cypress/report/mochawesome-report > cypress/report/mochawesome.json && ps -ef | grep docusaurus-start | grep -v grep | awk '{print $2}' | xargs kill
2020-09-16T05:59:54.226Z [INFO]: # Completed phase: postTest
2020-09-16T05:59:54.226Z [ERROR]: !!! Test execution failed, one or more tests are not passing
2020-09-16T05:59:54.227Z [INFO]: # Starting test artifact upload process...
2020-09-16T05:59:54.237Z [INFO]: # Uploading build artifact 'testConfig.json'...
2020-09-16T05:59:54.241Z [INFO]: [0] docusaurus-start --no-watch exited with code SIGTERM
2020-09-16T05:59:54.241Z [WARNING]: npm ERR! code ELIFECYCLE
                                    npm ERR! errno 1
                                    npm ERR! @ start:nw: `concurrently "docusaurus-start --no-watch" "npx build-slides"`
                                    npm ERR! Exit status 1
                                    npm ERR!
                                    npm ERR! Failed at the @ start:nw script.
                                    npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
                                    npm ERR! A complete log of this run can be found in:
                                    npm ERR!     /root/.npm/_logs/2020-09-16T05_59_54_237Z-debug.log
2020-09-16T05:59:54.241Z [HELP]: Outputting the npm debug log
                                 0 info it worked if it ends with ok
                                 1 verbose cli [ '/root/.nvm/versions/node/v10.16.0/bin/node',
                                 1 verbose cli   '/root/.nvm/versions/node/v10.16.0/bin/npm',
                                 1 verbose cli   'run',
                                 1 verbose cli   'start:nw' ]
                                 2 info using npm@6.9.0
                                 3 info using node@v10.16.0
                                 4 verbose run-script [ 'prestart:nw', 'start:nw', 'poststart:nw' ]
                                 5 info lifecycle @~prestart:nw: @
                                 6 info lifecycle @~start:nw: @
                                 7 verbose lifecycle @~start:nw: unsafe-perm in lifecycle true
                                 8 verbose lifecycle @~start:nw: PATH: /root/.nvm/versions/node/v10.16.0/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/codebuild/output/src561133977/src/docs/website/node_modules/.bin:/root/.nvm/versions/node/v10.16.0/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/codebuild/output/src561133977/src/docs/website/node_modules/.bin:/usr/local/rvm/gems/ruby-2.4.6/bin:/usr/local/rvm/gems/ruby-2.4.6@global/bin:/usr/local/rvm/rubies/ruby-2.4.6/bin:/usr/local/rvm/bin:/root/.yarn/bin:/root/.config/yarn/global/node_modules/.bin:/root/.nvm/versions/node/v10.16.0/bin:/root/.local/bin:/usr/local/rvm/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
                                 9 verbose lifecycle @~start:nw: CWD: /codebuild/output/src561133977/src/docs/website
                                 10 silly lifecycle @~start:nw: Args: [ '-c',
                                 10 silly lifecycle   'concurrently "docusaurus-start --no-watch" "npx build-slides"' ]
                                 11 silly lifecycle @~start:nw: Returned: code: 1  signal: null
                                 12 info lifecycle @~start:nw: Failed to exec start:nw script
                                 13 verbose stack Error: @ start:nw: `concurrently "docusaurus-start --no-watch" "npx build-slides"`
                                 13 verbose stack Exit status 1
                                 13 verbose stack     at EventEmitter.<anonymous> (/root/.nvm/versions/node/v10.16.0/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:301:16)
                                 13 verbose stack     at EventEmitter.emit (events.js:198:13)
                                 13 verbose stack     at ChildProcess.<anonymous> (/root/.nvm/versions/node/v10.16.0/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
                                 13 verbose stack     at ChildProcess.emit (events.js:198:13)
                                 13 verbose stack     at maybeClose (internal/child_process.js:982:16)
                                 13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:259:5)
                                 14 verbose pkgid @
                                 15 verbose cwd /codebuild/output/src561133977/src/docs/website
                                 16 verbose Linux 4.14.193-113.317.amzn1.x86_64
                                 17 verbose argv "/root/.nvm/versions/node/v10.16.0/bin/node" "/root/.nvm/versions/node/v10.16.0/bin/npm" "run" "start:nw"
                                 18 verbose node v10.16.0
                                 19 verbose npm  v6.9.0
                                 20 error code ELIFECYCLE
                                 21 error errno 1
                                 22 error @ start:nw: `concurrently "docusaurus-start --no-watch" "npx build-slides"`
                                 22 error Exit status 1
                                 23 error Failed at the @ start:nw script.
                                 23 error This is probably not a problem with npm. There is likely additional logging output above.
                                 24 verbose exit [ 1, true ]
2020-09-16T05:59:54.449Z [INFO]: # Build artifact is: 4MB
2020-09-16T05:59:54.449Z [INFO]: # Uploading build artifact 'testArtifacts.zip'...
2020-09-16T05:59:54.701Z [INFO]: # Test artifact upload completed
2020-09-16T05:59:54.702Z [ERROR]: !!! Non-Zero Exit Code detected
2020-09-16T05:59:54.702Z [INFO]: # Starting environment caching...
2020-09-16T05:59:54.702Z [INFO]: # Environment caching completed
Terminating logging...

**Comment by prasanth-reddy-wavemaker-com** [comment]
Did you run the tests again? are the failures reported fixed? @Tarun-Dubey @rnaresh

File: learn/teams/overview.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: "Overview"
+id: ""
+---
+---
+
+**Teams** are groups of organization members that reflect your organization and organize your group’s work. Teams are designed and intended to be administered by one or more members. On a team's page, you can
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/teams/overview.md:7]: Needs rephrasing

**Review by prasanth-reddy-wavemaker-com — CHANGES_REQUESTED** [review]
@swethakundaram can you review this once and then i will start. Quick check reveals some of the articles needs simplification

**Comment by rnaresh** [comment]
> Did you run the tests again? are the failures reported fixed? @Tarun-Dubey @rnaresh

@Tarun-Dubey fixed the failures @prasanth-reddy-wavemaker-com

File: learn/teams/overview.md (hunk 1 above)
Comment (swethakundaram) [reply learn/teams/overview.md:7]: Working on it.

File: learn/teams/code-repository.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+---
+title: Code Repository
+id: ""
+---
+---
+
+Source code management is a critical part of the project lifecycle, allowing teams to manage their work with a single source of truth. Team includes support for integrating multiple source code repositories with the system so that you have access to source code and control over where your source code resides. It helps in setting up Continuous Integration (CI) and Continuous Delivery (CD) pipelines. You can add multiple code repositories where the particular project's source code will reside.
```
Comment (krishnaraoch) [inline learn/teams/code-repository.md:7]: Write about what happens when multiple code repos are configured. 
What happens if new code repo added and marked as default. for old projects and new projects.
What is default code repo and what are the limitations of it.
What is the difference between push to external repo and push from studio(many customers confusing on this)
Does every developer need to provide credentilas while pushing.. how is the commit history look like. You can attach some flow here.

File: learn/teams/manage-projects.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
+---
+title: Manage Projects
+id: ""
+---
+---
+
+One of the main features of **Team** is the ability to share projects and collaborate on them with other users. Sharing a project with team member is more powerful, as the only users who will be allowed to access the project are users who you have invited to collaborate on the project. You can even choose to restrict what type of access other users have to specific projects.
```
Comment (krishnaraoch) [inline learn/teams/manage-projects.md:7]: Add the importance of having the control here.  Ex: Take some usecase like a person leaving org.
How does it different from having the control on project admin(from Studio page).
To whom he/she can share - only inside the org. how he gets notified etc.

File: learn/teams/manage-roles.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
+---
+title: Manage Roles
+id: ""
+---
+---
+
+It helps define custom roles (e.g. resource managers/developers) to give fine-grained access privileges to projects. And assign user roles like resource managers/developers to the members of your team's projects. Customize the access permissions for each role as you want. A user can perform actions on a project based on their assigned role & permissions.
```
Comment (krishnaraoch) [inline learn/teams/manage-roles.md:7]: Give some breif about how does it  different from roles of users from User section.

File: learn/teams/manage-subscription.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+---
+
+You can delegate subscription administration to manage members, view the current subscription, members and your subscription expiration date.
+
+-  You can access the Manage Subscription page in the administration area by clicking on the **Manage Subscription** entry in the left panel.
+
+![Manage Subscription](/learn/assets/TeamManageSubscription.png)
+
+
+-  To Change the owner of subscription, click on **Change Owner** button, a pop-up will open, select the member to be made new subscription owner and click on **Save** button.
```
Comment (krishnaraoch) [inline learn/teams/manage-subscription.md:14]: What is the use case for change owner ?
Diffirentiage  between wavemaker users vs own team members

File: learn/teams/manage-users.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+You can block/unblock a user by using the **block/ublock** icon located in each row in the table
+
+![Add users](/learn/assets/TeamUserBlock.png)
+
+
+### Remove a User
+
+You can delete a user by using the **Delete** icon located in each row in the table
+
```
Comment (krishnaraoch) [inline learn/teams/manage-users.md:47]: Usecase for Block and Delete ?
when to do what ?
what happens internally when click on Block and click on Delete

**Review by krishnaraoch — CHANGES_REQUESTED** [review]

File: learn/teams/overview.md [hunk 7]
```diff
@@ -1,22 +1,41 @@
 ---
-title: Teams Overview
+title: "Overview"
 id: ""
 ---
 ---
 
-Teams are groups of organization members that reflect your organization and organize your group’s work. On a team's page, you can 
+**Teams** are groups of organization members that reflect your organization and organize your group’s work. Teams are designed and intended to be administered by one or more members. On a team's page, you can
```
Comment (krishnaraoch) [inline learn/teams/overview.md:7]: Can we SILO vs Team. Its basically not a different product. its differnt subscription model.
Link to Subscription in the Website ???
What he can't get ? Customer may expect he can access service behind vpn..  
Vs WME ? No Infra setup etc ?????

**Review by krishnaraoch — CHANGES_REQUESTED** [review]

**Review by krishnaraoch — APPROVED** [review]

**Review by swethakundaram — APPROVED** [review]

File: learn/teams/overview.md (hunk 1 above)
Comment (swethakundaram) [reply learn/teams/overview.md]: You can review it now. @prasanth-reddy-wavemaker-com

**Pull request merged**